### PR TITLE
[TEST] LCP & LOAD timings

### PR DESCRIPTION
**Pull request by leeoniya**

### CODE REVIEW

File: public/app/app.ts [hunk 1]
```diff
@@ -106,6 +106,19 @@ import { createSystemVariableAdapter } from './features/variables/system/adapter
 import { createTextBoxVariableAdapter } from './features/variables/textbox/adapter';
 import { configureStore } from './store/configureStore';
 
+const observer = new PerformanceObserver((list) => {
+  const entries = list.getEntries();
+  const lastEntry = entries[entries.length - 1]; // Use the latest LCP candidate
+  console.log("LCP:", lastEntry.startTime);
+  // console.log(lastEntry);
```
Comment (baldm0mma) [inline public/app/app.ts:113]: Is this needed?

**Review by baldm0mma — APPROVED** [review]
LCP metrics, yay!!!

**Comment by leeoniya** [comment]
not meant for merging, just wanted to to get some measurements for this on HG via the ephemeral instance :)

**Comment by baldm0mma** [comment]
Got it. 👍

**Pull request closed without merging**